features/Makefile

```diff
@@ -11,7 +11,9 @@ ginkgo:
 	go get github.com/onsi/ginkgo/ginkgo
 
 functests: ginkgo
-	GOFLAGS=-mod=vendor ginkgo functests -- -junit /tmp/artifacts/unit_report.xml
+	FOCUS=$$(echo $(FEATURES) | tr ' ' '|') && \
+	echo "Focusing on $$FOCUS" && \
```

Comment:
it would be nice if you provide in the commit message a few examples about how it should work.

Reply:
My idea was to provide a README file, but will also add that to the commit message.

Reply:
Done